compte/views.py

```diff
@@ -149,6 +154,30 @@ def change_email(request, activation_token):
         )
 
 
+@login_required
+def delete_account(request):
+    try:
+        res = remove_personal_informations(request.user, shortuuid.uuid)
+    except RuntimeError as err:
+        messages.add_message(
+            request,
+            messages.WARNING,
+            f"Erreur lors de la désactivation du compte: {err}",
+        )
+        return redirect("mon_compte")
+
+    if res is True:
+        logout(request)
```

Comment:
The return value of `remove_personal_informations` is always `True`, is this check necessary?